docs/source/_topic_map.yml

```diff
@@ -14,8 +14,8 @@ Topics:
         File: installing
       - Name: Quickstart
         File: quickstart
-      - Name: Installing Docker Machine drivers
-        File: docker-machine-drivers
+      - Name: Setting up the hypervisor driver plugin
+        File: setting-up-hypervisor-driver-plugin
```

Comment:
@LalatenduMohanty +1 on my comment, @budhrg +1 on this change, and I was in favor from the beginning... (3-1 in favor of the change ?)